docs/source/jit_language_reference.rst

```diff
@@ -0,0 +1,438 @@
+     - Not Relevant
+     -
+   * - `4.2. Naming and binding <https://docs.python.org/3/reference/executionmodel.html#naming-and-binding>`_
+     - Not Relevant
+     - Names are entriely bound at compile time in TorchScript
+   * - `4.2.1. Binding of names <https://docs.python.org/3/reference/executionmodel.html#binding-of-names>`_
+     - Not Relevant
+     - See ``global`` and ``nonlocal`` statements
+   * - `4.2.2. Resolution of names <https://docs.python.org/3/reference/executionmodel.html#resolution-of-names>`_
+     - Not Relevant
+     - See ``global`` and ``nonlocal`` statements
+   * - `4.2.3. Builtins and restricted execution <https://docs.python.org/3/reference/executionmodel.html#builtins-and-restricted-execution>`_
+     - Not Relevant
+     -
+   * - `4.2.4. Interaction with dynamic features <https://docs.python.org/3/reference/executionmodel.html#interaction-with-dynamic-features>`_
+     - Not Supported
+     - Python values cannot be captured so this is not an issue
+   * - `4.3. Exceptions <https://docs.python.org/3/reference/executionmodel.html#exceptions>`_
+     - Partially Supported
+     - See ``try`` and ``raise`` statement
+   * - `5. The import system <https://docs.python.org/3/reference/import.html>`_
+     - Not Relevant
+     -
+   * - `6. Expressions <https://docs.python.org/3/reference/expressions.html#>`_
+     - Not Relevant
+     -
+   * - `6.1. Arithmetic conversions <https://docs.python.org/3/reference/expressions.html#arithmetic-conversions>`_
+     - Partially Supported
+     - Complex number coercion doesn't work, but (float + int) -> float does work
+   * - `6.2. Atoms <https://docs.python.org/3/reference/expressions.html#atoms>`_
+     - Not Relevant
+     -
+   * - `6.2.1. Identifiers (Names) <https://docs.python.org/3/reference/expressions.html#atom-identifiers>`_
+     - Supported
+     -
+   * - `6.2.2. Literals <https://docs.python.org/3/reference/expressions.html#literals>`_
+     - Partially Supported
+     - bytesliteral, imagnumber not supported
+   * - `6.2.3. Parenthesized forms <https://docs.python.org/3/reference/expressions.html#parenthesized-forms>`_
+     - Supported
+     -
+   * - `6.2.4. Displays for lists, sets and dictionaries <https://docs.python.org/3/reference/expressions.html#displays-for-lists-sets-and-dictionaries>`_
+     - Partially Supported
+     - Not supported: comprehension ifs, async iterators
+   * - `6.2.5. List displays <https://docs.python.org/3/reference/expressions.html#list-displays>`_
+     - Supported
+     -
+   * - `6.2.6. Set displays <https://docs.python.org/3/reference/expressions.html#set-displays>`_
+     - Not Supported
+     -
+   * - `6.2.7. Dictionary displays <https://docs.python.org/3/reference/expressions.html#dictionary-displays>`_
+     - Supported
+     - dict() constructor with kwargs doesn't work, dict comprehensions, dictionary unpacking, limited key types (hashable TorchScript classes??)
```

Comment:
maybe instead: `not supported: dict() constructor with kwargs, dict comprehensions, ...`. Maybe rephrase or remove `limited key types (hashable TorchScript classes??)`